config/tutorials/hdfs-logs/searcher-2.yaml

```diff
@@ -1,8 +1,7 @@
 version: 0
 node_id: searcher-2
-searcher:
-  rest_listen_address: 127.0.0.1
-  rest_listen_port: 7290
-  peer_seeds:
-    - 127.0.0.1:7280 # searcher-1
-    - 127.0.0.1:7300 # searcher-3
+listen_address: 127.0.0.1
+rest_listen_port: 7290
+seeds:
```

Comment:
nitpick: `peer_seeds` sound more into our tradition of being clear in naming.

Reply:
Yes, I like `peer_seeds` better too.

Reply:
Ok I will make a concession with time, but be aware that it cost me a lot.